src/core/helpers.js

```diff
@@ -1252,40 +1252,59 @@ wb.escapeAttribute = function( str ) {
 
 /*
 * Find most common Personal Identifiable Information (PII) in a string and return either the cleaned string either true/false
-* @param {string} str
-* @param {boolean} toClean
+* @param {object} object having the following properties:
+* 	{string} str
+* 	{boolean} toClean (optional)
+* 	{boolean} bMarker (optional) - if true, will replace the scrubbed values with string "[REDACTED/CAVIARDÉ]"
+* 	{string} caseOnly (optional) - this param should match a key name from the "regEx" object. If present, it will search and replace only the values corresponding that pattern
+* 	{regex} customCase (optional) - this param is a regex. It will search and replace the values corresponding that pattern
+
 * @return {string | true | false}
 * @example
-* wb.findPotentialPII( "email:test@test.com, phone:123 123 1234", true )
+* wb.findPotentialPII( { "str":"email:test@test.com, phone:123 123 1234", "toClean":true } )
 * returns "email:, phone:",
-* wb.findPotentialPII( "email:test@test.com, phone:123 123 1234", false )
+*
+* wb.findPotentialPII( { "str":"email:test@test.com, phone:123 123 1234", "toClean":false } )
 * returns true
+*
+* wb.findPotentialPII( { "str":"email:test@test.com, phone:123 123 1234", "toClean":true, "bMarker":true, "caseOnly":"email" } )
+* returns "email:[REDACTED/CAVIARDÉ], phone:123 123 1234"
+*
+* wb.findPotentialPII( { "str":"email:test@test.com, phone:123 123 1234, numéro de cas 12345678", "toClean":true, "bMarker":true, "customCase":/\b(?:case[\s-]?number[\s\-\\.]?(?:\d{5,10}))|(?:numéro[\s-]?de[\s-]?cas[\s\-\\.]?(?:\d{5,10}))/ig })
+* returns "email:[REDACTED/CAVIARDÉ], phone:[REDACTED/CAVIARDÉ], [REDACTED/CAVIARDÉ]"
 */
-wb.findPotentialPII = function( str, toClean ) {
-
-	if ( typeof str  !== "string" ) {
+wb.findPotentialPII = function( args ) {
+	if ( args.str && typeof args.str  !== "string" ) {
```

Comment:
We will need to avoid the breaking change. You will need a way to support the old way to sending the parameter.

Reply:
Fixed! Now the function accepts 2 params:

1. str  - the content to be verified;
2.  opts that can be passed: 
- as Boolean:  if set as true, it will simply scrub the data (this will avoid the major change);
- as object: here can be passed the other scrubbing options (bMarker, caseOnly, customCase)